packages/starkiller/PKGBUILD

```diff
@@ -0,0 +1,42 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=starkiller
+pkgver=v1.10.0.r0.g16d4e2f
+pkgrel=1
+pkgdesc='A frontend for PowerShell Empire.'
+arch=('x86_64')
+groups=('blackarch' 'blackarch-exploitation' 'blackarch-backdoor'
+        'blackarch-binary')
+url='https://github.com/BC-SECURITY/Starkiller'
+license=('MIT')
+depends=('nodejs')
+makedepends=('git' 'yarn' 'npm')
+options=(!strip)
+source=("git+https://github.com/BC-SECURITY/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
+}
+
+package() {
+  cd $pkgname
+
+  _npmdir="$pkgdir/usr/lib/node_modules/"
+
+  install -dm 755 "$_npmdir"
+
+  cd "$_npmdir"
+
+  npm install --user root -g --prefix "$pkgdir/usr" "$pkgname@"
```

Comment:
You used `PKGBUILD-nodejs-npm` template but starkiller is not available on npm so it won't work, please test your packages before submitting them.

```
==> Starting package()...
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/starkiller - Not found
npm ERR! 404 
npm ERR! 404  'starkiller@*' is not in this registry.
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /build/.npm/_logs/2022-07-10T15_44_40_798Z-debug-0.log
==> ERROR: A failure occurred in package().
    Aborting...
==> ERROR: Build failed, check /home/noraj/blackarch_chroot/blackarch/build
Failed to build the package
```